workers/unity/Assets/Playground/Scripts/EntityInitialization/ArchetypeInitializationSystem.cs

```diff
@@ -9,7 +9,7 @@ namespace Playground
     /// <summary>
     ///     Optionally adds a list of components to newly spawned entities according to an archetype definition.
     /// </summary>
-    [UpdateInGroup(typeof(SpatialOSReceiveGroup.EntityInitialisationGroup))]
+    [UpdateInGroup(typeof(SpatialOSUpdateGroup))]
```

Comment:
Doesn't this technically need to run before GameObjects are created? Since this system could potentially add more components that users could rely on during the gameobject creation process?

Reply:
How would they rely on those ECS components in the GameObject context? If they're writing a custom system to replace our GameObjectInitializationSystem they can 1) have that run after the Archetype stuff, 2) customize the Archetype stuff too. If they don't, they wouldn't have access to those components anyway right?

Reply:
this system is a playground system anyway. If users want to do that, they can create their own system and run it in the correct update loop. Right now, we are not depending on it so I don't see a reason to have it anything but the `SpatialOSUpdateGroup`

Reply:
Even in playground this is under constrained. Users need to be able able to rely on the new components being added while reading the initial component updates. This may eventually cause us odd bugs.

Reply:
I guess we can just change it to being updates before `SpatialOSUpdateGroup`?